test/new_relic/agent/llm/chat_completion_summary_test.rb

```diff
@@ -62,12 +62,15 @@ def test_record_creates_an_event
         summary.duration = '500'
         summary.error = 'true'
         summary.llm_version = '2022-01-01'
-        summary.rate_limit_requests = '100'
-        summary.rate_limit_tokens = '101'
+        summary.rate_limit_requests = 100
+        summary.rate_limit_tokens = 101
         summary.rate_limit_reset_tokens = '102'
         summary.rate_limit_reset_requests = '103'
-        summary.rate_limit_remaining_tokens = '104'
-        summary.rate_limit_remaining_requests = '105'
+        summary.rate_limit_remaining_tokens = 104
+        summary.rate_limit_remaining_requests = 105
+        summary.rate_limit_limit_tokens_usage_based = '106'
```

Comment:
Why are some strings and some ints

Reply:
It's just to follow what's in the spec. I think some of the headers, like 
ratelimitResetTokens, have values that are a mix of letters and numbers. I was hoping NRDB could do the transformation for us, but unfortunately that's not how it works.

Reply:
Would you like me to update the test data to be more representative of what the actual response headers would contain? the examples in the `response_headers_test` are more precise.

Reply:
I went ahead and did this anyway. I think it's better for documentation's sake to have something genuine. I looked at Python's tests to get the values added in e5450ba59